### PR TITLE
apt*, dpkg*: add Italian translation

### PR DESCRIPTION
**Pull request by Agno94**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

In `apt-get.md`, line 22, I translated `newest` as if it was `latest` because `ultima` sounds better then `più nuova` in Italian.

### CODE REVIEW

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks, @Agno94 :slightly_smiling_face:

/cc @glenacota, who IIRC speaks Italian

File: pages.it/linux/apt-get.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# apt-get
+
+> Servizio di gestione dei pacchetti per Debian e Ubuntu
+> Cerca i pacchetti usando `apt-cache`.
+
+- Aggiorna la lista dei pacchetti e delle loro versioni disponibili (prima di altri comandi `apt-get` è consigliato eseguire questo comando):
```
Comment (glenacota) [inline pages.it/linux/apt-get.md:6]: (minor, for readability) Consider using "è consigliato eseguire questo comando prima di altri comandi `apt-get`"

File: pages.it/linux/apt-get.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Pulisce gli archivi locali - rimuovendo i file (.deb) da scaricamenti interrotti che non possono più essere scaricati:
+
+`apt-get autoclean`
+
+- Rimuove tutti i pacchetti che non sono più necessari:
+
+`apt-get autoremove`
+
+- Aggiorna tutti i pacchetti installati(come `upgrade`), ma rimuove pure i pacchetti obsoleti ed installa ulteriori pacchetti per soddisfare le nuove dipendenze:
```
Comment (glenacota) [inline pages.it/linux/apt-get.md:34]: Please add whitespace before the parenthesis.
Also, consider replacing the subordinate with "[...], rimuovendo i pacchetti obsoleti e installando ulteriori pacchetti per soddifare le nuove dipendenze"

File: pages.it/linux/apt-key.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# apt-key
+
+> Servizio di gestione delle chiavi per il gestore di pacchetti APT su Debian ed Ubuntu.
+
+- Elenca le chiavi fidate:
```
Comment (glenacota) [inline pages.it/linux/apt-key.md:5]: Not sure whether "fidate" is the right translation here. I would suggest using "attendibile"

File: pages.it/linux/apt-key.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+- Aggiunge una chiave al portachiavi delle chiavi fidate:
+
+`apt-key add {{file_chiave_pubblica.asc}}`
+
+- Elimina una chiave dal portachiavi delle chiavi fidate:
+
+`apt-key del {{id_chiave}}`
+
+- Aggiunge una chiave remote al portachiavi delle chiavi fidate:
```
Comment (glenacota) [inline pages.it/linux/apt-key.md:17]: I think it's safe to use "chiave remota" here :)

File: pages.it/linux/apt-get.md (hunk 1 above)
Comment (glenacota) [reply pages.it/linux/apt-get.md:6]: the same recommendation applies also to "apt"

File: pages.it/linux/apt.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- Installa un pacchetto, o lo aggiorna all'ultima versione disponibile:
+
+`sudo apt install {{pacchetto}}`
+
+- Rimuove un pacchetto (usando `purge` rimuove anche i suoi file di configurazione):
+
+`sudo apt remove {{pacchetto}}`
+
+- Aggiorna tutti i pacchetti installati alla più nuova versione disponibile:
```
Comment (glenacota) [inline pages.it/linux/apt.md:26]: Consider replacing "nuova" with "recente"

**Comment by glenacota** [comment]
Great translations, @Agno94, thanks! Please have a look at my comments, and let me know what you think :)

File: pages.it/linux/apt-key.md (hunk 3 above)
Comment (Agno94) [reply pages.it/linux/apt-key.md:5]: Not sure either. `Trusted certificates` are translated to `certificati attendibili` for sure.
But I checked the Italian translation of seahorse's(the gnome application) GUI, and it uses `Mostra fidati`.
Another word could be `affidabili`.

Comment (Agno94) [reply pages.it/linux/apt-key.md:5]: I checked the Italian version of `apt-key` manpages and it uses "fidati"

Comment (glenacota) [reply pages.it/linux/apt-key.md:5]: good to know, thanks for checking!

**Review by glenacota — APPROVED** [review]
LGTM :)

**Pull request merged**